app/Http/Controllers/UpdatePersonalInfo.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\UpdatePersonalInfoRequest;
+use App\Transformers\UserTransformer;
+use Illuminate\Contracts\Routing\ResponseFactory;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Response;
+
+class UpdatePersonalInfo extends Controller
+{
+    public function __invoke(UpdatePersonalInfoRequest $request): JsonResponse|ResponseFactory
+    {
+        $user = $request->user();
+        $user->update($request->validated());
+        return responder()->success($user, UserTransformer::class)->respond(Response::HTTP_OK);
```

Comment:
When the response status is 200, because that's the default status code used, I don't see the point in explicitly indicating that code should be used.

Reply:
I thought it was better to explicitly show what I am trying to return and not leave it to default, but I will remove it then

Reply:
Not in my opinion. It's not necessarily "bad" to do this, but I don't see a real reason to do it.

Reply:
Oh okay I see what you meant now